src/SDKs/_metadata/sql_resource-manager.txt

```diff
@@ -1,11 +1,13 @@
-2018-01-31 10:51:15 UTC
+
```

Comment:
Please pull down the latest changes from upstream and run the `generate.cmd` command again, this should log the commit id here.

Reply:
@dsgouda .  I did, but the interesting thing is no commit id shown in the file. I tried several times.
 my forked azurePayi/azure-sdk-for-net repo is updated with the official azure/azure-sdk-for-net repo, and I run the generate.cmd against official azure/azure-rest-spi-specs
since my swagger changes already were merged to the official azure-rest-spi-specs

Reply:
@dsgouda Thanks for your help! Update the PR which has commit id in the metadata file.